Cargo.toml

```diff
@@ -1,7 +1,11 @@
 [package]
 name = "webex"
-version = "0.7.0"
-authors = ["Scott Hutton <shutton@pobox.com>", "Milan Stastny <milan@stastnej.ch>", "Abel Shields <abel@uucp.org.uk>"]
+version = "0.7.2"
```

Comment:
Not a biggie, but shouldn't we bump the minor to 8 here instead?

Reply:
You are right. I realised after the fact that the struct and its fields were all public. Let's see what you guys want to do with #21 and then I can change this to 0.8.0